tools/sof-dump-status.py

```diff
@@ -11,13 +11,23 @@ def __init__(self):
         self.acpi_lst=[]
         self.proc_card={}
         # https://github.com/thesofproject/linux/blob/topic/sof-dev/sound/soc/sof/sof-pci-dev.c
-        self._pci_ids={"0x119a":"tng","0x5a98":"apl", "0x1a98":"apl", "0x3198":"glk",
-            "0x9dc8":"cnl", "0xa348":"cfl", "0x9d71":"kbl", "0x9d70":"skl", "0x34c8":"icl", "0x38c8":"jsl", "0x4dc8":"jsl",
-            "0x02c8":"cml", "0x06c8":"cml", "0xa0c8":"tgl", "0x43c8":"tgl-h", "0x4b55":"ehl", "0x4b58":"ehl", "0x7ad0": "adl-s",
-            # https://github.com/thesofproject/linux/blob/topic/sof-dev/sound/soc/sof/sof-acpi-dev.c
-            "0x3438":"bdw", "0x33c8":"hsw", "0x0f04":"byt", "0x2284":"cht",
-            # https://github.com/thesofproject/linux/blob/topic/sof-dev/sound/pci/hda/hda_intel.c
-            "0x160c":"bdw" }
+        self._pci_ids={
+                #/linux/sound/soc/sof/intel/pci-tng.c
+                "0x119a":"tng",
+                #/linux/sound/soc/sof/intel/pci-apl.c
+                "0x5a98":"apl", "0x1a98":"apl", "0x3198":"glk",
+                #/linux/sound/soc/sof/intel/pci-cnl.c
+                "0x9dc8":"cnl", "0xa348":"cfl", "0x02c8":"cml", "0x06c8":"cml",
+                #/linux/sound/soc/sof/intel/pci-icl.c
+                "0x34c8":"icl", "0x38c8":"jsl","0x4dc8":"jsl",
+                #/linux/sound/soc/sof/intel/pci-tgl.c
+                "0xa0c8":"tgl", "0x43c8":"tgl-h", "0x4b55":"ehl", "0x4b58":"ehl","0x7ad0": "adl-s","0x51c8":"adl-p",
```

Comment:
The ADL-P platform is using `tgl_desc`, which define the FW name as `sof-tgl.ri`, so you should add `0x51c8` as tgl platform, rather than adl-p. @plbossart Do we have plan to add a new adlp_desc for adl-p platform?
@XiaoyunWu6666 Before you make any code change, let's wait  Pierre's feedback.
```
	{ PCI_DEVICE(0x8086, 0x51c8), /* ADL-P */
		.driver_data = (unsigned long)&tgl_desc},
```

Reply:
Please revise the comment,
`#/linux/sound/soc/sof/intel/pci-tgl.c` -> `https://github.com/thesofproject/linux/blob/topic/sof-dev/sound/soc/sof/intel/pci-tgl.c`

Then we can view the linux file by a mouse click.

Reply:
I find this very verbose and repetitive. I think having a full URL a single time at the top is enough,  two clicks does not take much time and code is read much more often than clicked on.

Reply:
new descriptors were added today for ADL in the topic/sof-dev branch